src/OpenTelemetry.Exporter.OneCollector/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 1.5.1-rc.1
```

Comment:
Maybe change this to beta? Are we going to do a stable release soon?

Reply:
Plan is to release stable early August.